examples/webserver.rs

```diff
@@ -66,13 +65,13 @@ fn main() -> Result<()> {
             response.write_all("no framebuffer".as_bytes())?;
         }
 
-        Ok(())
+        Ok::<(), anyhow::Error>(())
```

Comment:
why?

Reply:
It won't compile otherwise, I believe a change in the upstream api